src/module/actor/familiar/sheet.ts

```diff
@@ -30,9 +30,11 @@ export class FamiliarSheetPF2e<TActor extends FamiliarPF2e> extends CreatureShee
     override async getData(options?: ActorSheetOptions): Promise<FamiliarSheetData<TActor>> {
         const sheetData = await super.getData(options);
         const familiar = this.actor;
-        // Get all potential masters of the familiar
+
+        // Get all potential masters of the familiar (always include current master regardless of User permissions)
         const masters = game.actors.filter(
-            (a): a is CharacterPF2e<null> => a.type === "character" && a.testUserPermission(game.user, "OWNER"),
+            (a): a is CharacterPF2e<null> =>
+                (a.type === "character" && a.testUserPermission(game.user, "OWNER")) || this.actor.master?.id === a.id,
```

Comment:
Looks like the parentheses are grouping the wrong conjuncts: it should always be a PC

Reply:
That was intentional but a mistake born of ignorance. 😅 Fixed